composites/Plugin/Shared/components/KeywordInput.js

```diff
@@ -78,11 +94,10 @@ class KeywordInput extends React.Component {
 	 *
 	 * @param {Event} event The onChange event.
```

Comment:
we're documenting React event in a inconsistency way, technically they're not DOM events, but "SyntheticEvent". In a few places we do use `{object} event React SyntheticEvent`.